docs/content/docs/updates/v1.0.0.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Fluid Framework 1.0
+aliases:
+  - /updates/1.0/
+summary: |
+  The 1.0 release is the first major release of the Fluid Framework.
+---
+
+## Breaking changes
+
+### connected() on FluidContainer is replaced with connectionState
+
+The `connected()` method has been replaced with the `connectionState` property. Read more
```

Comment:
Technically this is a method since it's a getter, but it essentially acts as a boolean for end users, so I think it makes sense to call it that here.
```suggestion
The `connected` boolean has been replaced with the `connectionState` property. Read more
```